osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/k8s/deploymentspec/ConformK8sDeploymentPodsMetaTask.java

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.service.tasks.conformance.k8s.deploymentspec;
+
+import java.util.Set;
+
+import javax.persistence.EntityManager;
+
+import org.osc.core.broker.job.TaskGraph;
+import org.osc.core.broker.job.lock.LockObjectReference;
+import org.osc.core.broker.model.entities.virtualization.openstack.DeploymentSpec;
+import org.osc.core.broker.service.tasks.TransactionalMetaTask;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * Conforms the deployment pods for a Kubernetes deployment.
+ *
+ */
+@Component(service = ConformK8sDeploymentPodsMetaTask.class)
+public class ConformK8sDeploymentPodsMetaTask extends TransactionalMetaTask {
+
+
+    private DeploymentSpec ds;
+    private TaskGraph tg;
+
+    public ConformK8sDeploymentPodsMetaTask create(DeploymentSpec ds) {
+        ConformK8sDeploymentPodsMetaTask task = new ConformK8sDeploymentPodsMetaTask();
+        task.ds = ds;
+
+        return task;
+    }
+
+    @Override
+    public void executeTransaction(EntityManager em) throws Exception {
+        this.tg = new TaskGraph();
```

Comment:
Unimplemented, I presume

Reply:
Yes, it will be implemented by a future, soon to come, PR. I dont want to throw an exception at the moment because I want to see if the jobs pass right now. This task is added to a parent graph so if i throw it will fail the graph.